### PR TITLE
Add AUDIO_CLASH_SUPPRESSION_LEVEL

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: props/prop_base.h [hunk 1]
```diff
@@ -17,6 +17,11 @@
 #undef SAVE_VOLUME
 #endif
 
+#ifndef AUDIO_CLASH_SUPPRESSION_LEVEL
+// Account for Audio Volume in Clash Detection (range 1 ~ 50)
+#define AUDIO_CLASH_SUPPRESSION_LEVEL 1
```
Comment (profezzorn) [inline props/prop_base.h:22]: If the default value is this low, is there really any point?

File: props/prop_base.h [hunk 2]
```diff
@@ -721,9 +726,9 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     } else {
       v = diff.len();
     }
-    // If we're spinning the saber, require a stronger acceleration
-    // to activate the clash.
-    if (v > CLASH_THRESHOLD_G + fusor.gyro().len() / 200.0) {
+    // If we're spinning the saber or if loud sounds are playing, 
+    // require a stronger acceleration to activate the clash.
+    if (v > (CLASH_THRESHOLD_G + fusor.gyro().len() / 200.0) + (dynamic_mixer.audio_volume() * (AUDIO_CLASH_SUPPRESSION_LEVEL * 0.000001))) {
```
Comment (profezzorn) [inline props/prop_base.h:731]: This needs some line-breaking to make the line shorter.
(Probably after the plusses)

File: props/prop_base.h (hunk 1 above)
Comment (fdarosa2663) [reply props/prop_base.h]: I was originally thinking you'd want it to be almost nothing if the define was missing to keep existing behavior from previous versions

File: props/prop_base.h (hunk 2 above)
Comment (fdarosa2663) [reply props/prop_base.h]: added line break

**Pull request merged**